README.md

```diff
@@ -1 +1,62 @@
-# WEB-SUBMIT-2022-ACCRA
\ No newline at end of file
+# Web Summit 2022 Accra
+
+Live preview [here](https://jerryowusu.github.io/Web-Summit-Accra/). 
+
+## Screenshots
+![Example screenshot](/src/images/desktop-screenshot.png)
+
+## Contents
+* [Homepage](#homepage)
+* [Program](#program)
+* [Partners](#partners)
+* [Contact](#contact)
+* [Mobile menu](#mobile)
+* [Click more for expand speakers](#speakers)
+
+## General info
+This project is my first Capstone.
+	
+## Technologies
+Project is created with:
+* HTML
+* CSS
+* Bootstrap
+* Javascript
+
+	
+## Setup
+To run this project, install it locally using:
+$ cd Desktop
+$ git clone https://github.com/jerryowusu/Web-Summit-Accra.git
```

Comment:
- [ ] Kindly add a description of the project you have worked on to your readme file.
- [ ] It would be better to add more details on how to use your project locally.

Reply:
Noted with thanks